python/benchmark/databricks/init-pip-cuda-11.8.sh

```diff
@@ -24,12 +24,17 @@ ln -s /usr/local/cuda-11.8 /usr/local/cuda
 
 # install cudf and cuml
 # using ~= pulls in micro version patches
-/databricks/python/bin/pip install cudf-cu11~=${RAPIDS_VERSION} \
-    cuml-cu11~=${RAPIDS_VERSION} \
-    cuvs-cu11~=${RAPIDS_VERSION} \
-    pylibraft-cu11~=${RAPIDS_VERSION} \
-    rmm-cu11~=${RAPIDS_VERSION} \
-    --extra-index-url=https://pypi.nvidia.com
+# /databricks/python/bin/pip install cudf-cu11~=${RAPIDS_VERSION} \
+#     cuml-cu11~=${RAPIDS_VERSION} \
+#     cuvs-cu11~=${RAPIDS_VERSION} \
+#     pylibraft-cu11~=${RAPIDS_VERSION} \
+#     rmm-cu11~=${RAPIDS_VERSION} \
+#     --extra-index-url=https://pypi.nvidia.com
+
+/databricks/python/bin/pip install \
+    --extra-index-url=https://pypi.anaconda.org/rapidsai-wheels-nightly/simple \
+    "cudf-cu11>=24.10.0a0,<=24.10" "dask-cudf-cu11>=24.10.0a0,<=24.10" \
+    "cuml-cu11>=24.10.0a0,<=24.10" "dask-cuda>=24.10.0a0,<=24.10"
```

Comment:
Would cuvs be automatically installed even it is not specified in the pip install command?
Seems we introduce cudf, dask-cudf, and dask-cuda to the pip install command. Any reason for that?